lib/runtime/wasmer/instance.go

```diff
@@ -193,13 +193,7 @@ func NewInstance(code []byte, cfg Config) (*Instance, error) {
 	}
 
 	if cfg.testVersion != nil {
-		instance.ctx.Version = *cfg.testVersion
-	} else {
-		instance.ctx.Version, err = instance.version()
```

Comment:
So to understand, we are avoiding setting the version here to push the installation to the first Version() call, where it can then be cached? Am I understanding this correct?

Reply:
yes, that's correct

Reply:
Does it get refreshed during the runtime upgrades?

Reply:
an instance is an instance of a particular runtime. You in case of a runtime upgrade, we will be dealing with a new instance.